delete_files_week_older.sh

```diff
@@ -0,0 +1,6 @@
+# to run as cronjob every night
+
+DAY_LAPSE=7
+
+# to test scrpt remove -delete in the end
+find $APARTMENT_DATA_TRANSFER_PATH -maxdepth 1 -name "*.xml" -mtime +$DAY_LAPSE -type f -print -delete
```

Comment:
It could be good to check in if-clause to that $APARTMENT_DATA_TRANSFER_PATH has been set and the folder exists. Otherwise, this run fails. Later here could come other files as well which needs to clean up too.

Reply:
the error is `find: ‘XXX’: No such file or directory`. What do we want to do in this case? some kind of logger message?

Reply:
I'm thinking if there could be an if-clause e.g. like this and it skips the command if the env hasn't been set or the folder doesn't exist:
```
if [ -n "${VAR}" ] && -d "${VAR}"; then
   find $APARTMENT_DATA_TRANSFER_PATH -maxdepth 1 -name "*.xml" -mtime +$DAY_LAPSE -type f -print -delete
fi
```